analysis/utility_analysis.py

```diff
@@ -59,16 +58,11 @@ def perform_utility_analysis(
       public_partitions: A collection of partition keys that will be present
         in the result. If not provided, the utility analysis with private
         partition selection will be performed.
-      return_per_partition: if true, in addition it returns utility analysis per
-        partitions as a 2nd element.
     Returns:
-         if return_per_partition == False:
-            returns collections which contains metrics.UtilityReport with
-            one report per each input configuration
-         else:
-            returns a tuple, with the 1st element as in first 'if' clause and
-            the 2nd a collection with elements
-            (partition_key, [metrics.PerPartitionMetrics]).
+        returns a tuple, with the 1st element is a collection which contains
```

Comment:
returns a tuple. Its 1st element is a collection which contains
          metrics.UtilityReport with one report per each input configuration. The 2nd element of the tuple is a collection with elements
          (partition_key, [metrics.PerPartitionMetrics]).

Reply:
Thanks!